src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -614,7 +641,13 @@ public string RoleClaimType
         /// property is true or false.
         /// </remarks>
         [DefaultValue(true)]
-        public bool ValidateLifetime { get; set; }
+        public bool ValidateLifetime
+        {
+            get;
+
+            [Obsolete(LogMessages.IDX10111, false)]
```

Comment:
We should call out this as a potential _breaking_ change  (if warnings are threated as errors), in the release notes.

Reply:
Draft notes are now published here https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases/tag/untagged-cc6fe1c38dd4b5bc70e2